### PR TITLE
commit updating changes to homepage and bio

### PR DESCRIPTION
**Pull request by khuetu**
Hero section with decorations: 
Screen Shot 2022-06-15 at 22 02 40

Update pictures for core values:
Screen Shot 2022-06-15 at 22 03 32

Update fields to biography:
Screen Shot 2022-06-15 at 22 02 27

### CODE REVIEW

File: frontend/src/components/About.js [hunk 1]
```diff
@@ -5,16 +5,41 @@ import { BACKEND_URL } from '../utils/api'
...
+        count += 1
+      }
+    }
+  return count
+}
+
 const Bio = ({profile, avatar}) => {
+  while (countString(profile.name, ' ') !== 1) {
+    profile.name = profile.name.slice(profile.name.indexOf(' ') + 1)
+  }
```
Comment (npnkhoi) [inline frontend/src/components/About.js:21]: Is this logic required by Figma em oi? Looks overkill.

File: frontend/src/components/Home.js [hunk 2]
```diff
@@ -44,7 +45,7 @@ const Mission = () => {
 const Value = ({value, description, img}) => {
     return (
         <div className="col-12 col-md-6 text-center">
-            <img src={img} height="148" alt=""></img>
+            <img src={img} height="148" alt="" class="rounded"></img>
```
Comment (npnkhoi) [inline frontend/src/components/Home.js:48]: From the preview, I can see that the images are not in the same aspect ratio. Would be nice to have same ratio tho.

**Review by npnkhoi — APPROVED** [review]
Looks good. Some minor comments only. If you can fix them quickly, please do. Otherwise, just go ahead and merge by tmr.

File: frontend/src/components/About.js (hunk 1 above)
Comment (nhomnhomm) [reply frontend/src/components/About.js:21]: This is the design on Figma íi anh
Screen Shot 2022-06-16 at 15 35 52

**Pull request merged**